src/test/java/com/hyunec/cosmicbaseball/acceptancetest/NormalBaseballLv1Test.java

```diff
@@ -1,18 +1,77 @@
 package com.hyunec.cosmicbaseball.acceptancetest;
 
+import com.hyunec.cosmicbaseball.entity.BaseballGame;
+import com.hyunec.cosmicbaseball.handler.StrikeHandler;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
 
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SpringBootTest
 class NormalBaseballLv1Test {
-    @DisplayName("타격 결과는 모두 같은 확률을 가집니다.")
+
+    @Autowired
+    private StrikeHandler strikeHandler;
+    @Autowired
+    private BaseballGame baseballGame;
+
     @Test
-    void t1() {
-        throw new RuntimeException("Not yet implemented");
+    @DisplayName("타격 결과는 모두 같은 확률을 가집니다.")
+    void testSwing() {
+
+        // Given
+        int numberOfSwings                = 10000; // 스윙 횟수
+        Map<String, Integer> resultCounts = new HashMap<>();
+        resultCounts.put("스트라이크", 0);
+        resultCounts.put("볼", 0);
+        resultCounts.put("안타", 0);
+
+        // When : 스윙 시뮬레이션
+        for (int i = 0; i < numberOfSwings; i++) {
+            String result = baseballGame.swing();
+            resultCounts.put(result, resultCounts.get(result) + 1);
+        }
+
+        // Then
+        int expectedCount = numberOfSwings / 3;
+        int tolerance     = (int) (numberOfSwings * 0.05); // 허용 오차: 5%
+
+        assertTrue(Math.abs(resultCounts.get("스트라이크") - expectedCount) <= tolerance,
+                "스트라이크 횟수가 허용 오차 범위 내에 있지 않습니다.");
+        assertTrue(Math.abs(resultCounts.get("볼") - expectedCount) <= tolerance,
+                "볼 횟수가 허용 오차 범위 내에 있지 않습니다.");
+        assertTrue(Math.abs(resultCounts.get("안타") - expectedCount) <= tolerance,
+                "안타 횟수가 허용 오차 범위 내에 있지 않습니다.");
```

Comment:
- assert message 를 별도로 관리하는 방법 좋습니다. 👍
    - 하지만 저는 string 을 따로 관리해야되는 귀찮음이 있어 라이브러리에서 제공되는 기본형을 쓰곤 합니다.
- 오차 범위를 넣는 방법도 재미있는 접근 이네요. 👍
    - fyi. 비결정적 테스트

---

```suggestion
        assertThat(resultOccurred.get("스트라이크")).isTrue();
        assertThat(resultOccurred.get("볼")).isTrue();
        assertThat(resultOccurred.get("안타")).isTrue();
```

junit 의 assertion 을 그대로 사용하는 지금 방식도 좋지만, 실무에서는 조금 더 확장된 DSL 들을 사용합니다.
- fyi. assertJ, Kluent, awaitility

Reply:
테스트 코드는 junit만 사용해봐서,,,, 
fyi. assertJ, kluent, awaitility, kotest  하나도 모르네요 저는 ㅠㅠ..... 꼭 공부해봐서 적용해보겠습니당.

Reply:
테스트 코드를 위한 라이브러리를 학습하는 것도 꽤 비용이 듭니다.
따라서 모두 하는 것 보다는 지금을 유지하거나, assertJ 정도만 적용하는 것을 권장 드립니다.